main.go

```diff
@@ -70,6 +76,40 @@ func main() {
 	}
 }
 
+//abHandler ... percentA is the percentage of request that handler 'a' is used
+func abHandler(a, b http.Handler, percentA int) http.Handler {
+	if percentA < 0 || percentA > 100 {
+		panic("Percent 'a' but be between 0 and 100")
```

Comment:
It seems a little brutal to panic here. If we do need to panic, shouldn't we do it earlier on?